### PR TITLE
Rewrite the code with two different tests.

### PR DESCRIPTION
**Pull request by vaishnavibhat**
test_bridgeCreate() and test_bridgeDelete()

Signed-off-by: Vaishnavi Bhat <vaishnavi@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by vaishnavibhat** [comment]
[root@ltcfleet2-lp3 net]# avocado run bridge.py -m bridge.py.data/bridge.yaml 
JOB ID     : ed11970bb536d510acdbd83b11451990ba594e1f
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-19T17.13-ed11970/job.log
 (1/2) bridge.py:Bridging.test_bridgeCreate;run-97c7: PASS (3.33 s)
 (2/2) bridge.py:Bridging.test_bridgeDelete;run-97c7: PASS (3.47 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 6.95 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-19T17.13-ed11970/results.html

[root@ltcfleet2-lp3 net]# avocado run bridge.py:Bridging.test_bridgeCreate -m bridge.py.data/bridge.yaml 
JOB ID     : 7935f88d4a3aa86d577c8af3694cb1718780db4d
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-19T17.13-7935f88/job.log
 (1/1) bridge.py:Bridging.test_bridgeCreate;run-97c7: PASS (3.31 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 3.44 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-19T17.13-7935f88/results.html

[root@ltcfleet2-lp3 net]# avocado run bridge.py:Bridging.test_bridgeDel -m bridge.py.data/bridge.yaml
JOB ID     : 3d0ff7cb22f7ef47ac85c959ace478226be65a9a
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2019-11-19T17.13-3d0ff7c/job.log
 (1/1) bridge.py:Bridging.test_bridgeDelete;run-97c7: PASS (3.50 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 3.62 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2019-11-19T17.13-3d0ff7c/results.html

File: io/net/bridge.py [hunk 1]
```diff
@@ -80,28 +67,22 @@ def test(self):
 
         self.check_failure('ip link set %s master br0' % self.host_interface)
         self.check_failure('ip addr flush dev %s' % self.host_interface)
-        self.check_failure('ip addr add %s broadcast %s dev br0' %
-                           (self.cidr, self.broadcast))
+        self.check_failure('ip addr add %s/%s dev br0' %
+                           (self.ipaddr, self.netmask))
```
Comment (narasimhan-v) [inline io/net/bridge.py]: Why is this needed if we are already doing `configure_network.set_ip()`

Comment (vaishnavibhat) [reply io/net/bridge.py]: In configure_network.set_ip() , we are only configuring the host interface. 
Assigning IP is done with the above method.

Comment (narasimhan-v) [reply io/net/bridge.py]: @vaishnavibhat Can you explain a bit more on the difference please ?

File: io/net/bridge.py [hunk 2]
```diff
@@ -80,28 +67,22 @@ def test(self):
...
         self.check_failure('ip link set br0 up')
-        if self.gateway:
```
[the remark this answers is not in the document]
Comment (vaishnavibhat) [reply io/net/bridge.py]: Code is working as expected without the gateway. hence avoiding an extra parameter.

File: io/net/bridge.py (hunk 1 above)
Comment (vaishnavibhat) [reply io/net/bridge.py]: in the yaml file, we enter the interface name , IP and netmask of the interface that is a part of the bridge interface. configure_network.set_ip() with create a cfg file for this, get the interface up and check if it is working fine. 
The later part of the code is used to create bridge, configure it and do a ping check and do not need a cfg file. 
configure_network utility is available only for configuring normal interface and not for other devices.

Comment (narasimhan-v) [reply io/net/bridge.py]: please enable it for bridge interfaces @vaishnavibhat

Comment (vaishnavibhat) [reply io/net/bridge.py]: Using the  configure_network.set_ip() requires some changes in the  configure_network.py utility. 
Working on it.

**Comment by smruti77** [comment]
this one handle on #1580
